src/lib/Ledger/hooks/connection.ts

```diff
@@ -84,18 +81,12 @@ export const useLedgerConnection = () => {
     );
 
     const connect = useCallback(
-        async (
-            profile: Contracts.IProfile,
-            coin: string,
-            network: string,
-            retryOptions?: Options,
-        ) => {
-            const coinInstance = profile.coins().set(coin, network);
-
-            if (!isLedgerTransportSupported()) {
-                handleLedgerConnectionError({ message: 'COMPATIBILITY_ERROR' }, coinInstance);
-                return;
-            }
+        async (_profile: Contracts.IProfile, _network: string, retryOptions?: Options) => {
+            // TODO enable check
+            // if (!isLedgerTransportSupported()) {
+            //     handleLedgerConnectionError({ message: 'COMPATIBILITY_ERROR' }, coinInstance);
+            //     return;
+            // }
```

Comment:
```suggestion
        async (_profile: Contracts.IProfile, _network: string, retryOptions?: Options) => {
             if (!isLedgerTransportSupported()) {
                 handleLedgerConnectionError({ message: 'COMPATIBILITY_ERROR' });
                 return;
             }
```

Does it have any issues if we call it without the coinInstance? I see it's no longer required.

Reply:
I know, I left ledger part of it out of this task and left `sdk-ledger` dependency to remove in a separate task.

Reply:
https://app.clickup.com/t/86dx35z6b